integration/setup/setup.go

```diff
@@ -158,8 +157,30 @@ func Resources(config Config) error {
 		if err != nil {
 			return microerror.Mask(err)
 		}
+	}
+
+	{
+		c := chartvalues.APIExtensionsAzureConfigE2EConfig{
+			Azure: chartvalues.APIExtensionsAzureConfigE2EConfigAzure{
+				CalicoSubnetCIDR: env.AzureCalicoSubnetCIDR(),
+				CIDR:             env.AzureCIDR(),
+				Location:         env.AzureLocation(),
+				MasterSubnetCIDR: env.AzureMasterSubnetCIDR(),
+				VPNSubnetCIDR:    env.AzureVPNSubnetCIDR(),
+				WorkerSubnetCIDR: env.AzureWorkerSubnetCIDR(),
+			},
+			ClusterName:               env.ClusterID(),
+			CommonDomain:              env.CommonDomain(),
+			CommonDomainResourceGroup: "godsmack",
```

Comment:
I am not sure what to do with this. Having installation specific defaults in the templates repo feels kind of wrong.

Reply:
Could installation specific logic be in `e2e-harness` or `e2esetup`?

Reply:
This should be an env var IMO as the CommonDomain is.